app/components/Lightbox/index.js

```diff
@@ -1,19 +1,43 @@
-import React, {Component} from 'react'
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import styles from './styles.css'
+import Button from 'components/Button'
 
 class Lightbox extends Component {
+  constructor(props) {
```

Comment:
Unless we're doing something special in the constructor we don't need to declare it.